content/en/docs/concepts/storage/volumes.md

```diff
@@ -178,6 +178,11 @@ See the [CephFS example](https://github.com/kubernetes/examples/tree/{{< param "
 
 {{< note >}}
 Kubernetes must be configured with the OpenStack cloud provider.
+
+Starting with Kubernetes v1.18, it is recommended to use the OpenStack cloud
+provider CSI driver directly rather than using the cinder volume plugin.
+
+See below for details.
```

Comment:
I'd leave this detail out. Issue #22521 tracks separating out the built in storage driver list details from the task steps to migrate each of them to CSI, and the work on _that_ issue is where I recommend doing this kind of cleanup.

Reply:
Thanks, I wasn't aware of that issue. I'll leave it to that.

I do hope we get that addressed soon though. I've had multiple conversations with OpenStack users that have been confused by seeing "cinder - deprecated". Without knowing the nuance of how things work in much detail, they see that and assume "oh, I won't be able to use cinder with kubernetes anymore".

Update coming.